feature-libs/product/configuration/src/variant-configurator/interactive-configuration.module.ts

```diff
@@ -0,0 +1,119 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { CmsConfig, ConfigModule, ProductModule } from '@spartacus/core';
+import {
+  CmsPageGuard,
+  ConfigAddToCartButtonComponent,
+  ConfigFormComponent,
+  ConfigGroupMenuComponent,
+  ConfigGroupTitleComponent,
+  ConfigPreviousNextButtonsComponent,
+  ConfigPriceSummaryComponent,
+  ConfigProductTitleComponent,
+  ConfigTabBarComponent,
+  ConfigurationMessageLoaderModule,
+  DefaultMessageConfig,
+  HamburgerMenuModule,
+  PageLayoutComponent,
+} from '@spartacus/storefront';
+
```

Comment:
Can we add an API comment that briefly explains what is this `InteractiveConfigurationModule`?

Reply:
Did this..